R/financial.R

```diff
@@ -17,13 +17,7 @@
 dm_financial <- function() {
```

Comment:
Let's expose `source` here:

```suggestion
dm_financial <- function(source = "relational.fit") {
```